pkg/podman/errors.go

```diff
@@ -0,0 +1,21 @@
+package podman
+
+import (
+	"fmt"
+)
+
+type PodmanNotFoundError struct {
+	err error
+}
+
+func NewPodmanNotFoundError(err error) PodmanNotFoundError {
+	return PodmanNotFoundError{err: err}
+}
+
+func (o PodmanNotFoundError) Error() string {
+	msg := "unable to access podman. Do you have podman client installed?"
```

Comment:
Another suggestion to update this message, which could again be misleading, IMO.
```suggestion
	msg := "unable to access podman. Do you have podman client installed and configured correctly?"
```